native/native_main.py

```diff
@@ -449,6 +1090,8 @@ def handleMessage(message):
         reply["content"] = ""
         reply["code"] = 0
 
+    # FIXME: "write" below currently overwrite the previous content
```

Comment:
I don't think this is a bug. If we ever need to append things to files I think we should add an `append()` function.

Reply:
Well, I personally don't like my configuration files being overwritten without a warning first :-)